legal-api/src/legal_api/services/permissions.py

```diff
@@ -28,6 +28,7 @@
 class ListFilingsPermissionsAllowed(str, Enum):
     """Define an enum for permissions checks."""
 
+    ADD_ENTITY_NO_AUTHENTICATION = 'ADD_ENTITY_NO_AUTHENTICATION'
```

Comment:
hmm. i don't think its correct either. This enum seems to be specific to filing permissions. Its better to be in ListActionsPermissionsAllowed (from Rajan's PR https://github.com/bcgov/lear/pull/3741/files#diff-aded57016dcbf8c9a8a0e8e2c87543f60cf2eeda674a32cfd01be70506398b36R61)

Reply:
Okay. My misunderstanding! I have added the same enum as Rajan and added the permission in it.